src/PanelTraits/AutoSet.php

```diff
@@ -27,6 +41,19 @@ public function setFromDb()
                 'values'     => [],
                 'attributes' => [],
             ];
+
+            if(!is_null($crudTypes) && isset($crudTypes[$field])) {
```

Comment:
I found that without the check it was returning an error saying it was $crudType[$field] was not found.